frontend/reducers.js

```diff
@@ -19,4 +19,5 @@ export default {
   user,
   githubBranches,
   notifications,
+  FRONTEND_CONFIG: (state = {}) => state,
```

Comment:
kind of a no-op or pass-through reducer. we have no actions to modify the `FRONTEND_CONFIG` state, nor do we want any.